### ISSUE TITLE
(4) Apply new homepage design: Desktop - Deadline 11/10

### ISSUE THREAD

**Issue by JessWaggs** [UCSCLibrary/dams_project_mgmt#279]
### Descriptive Summary

As an end user, I want to know what I might find in a "Digital Collections," have a readily apparent way to search for items, and have the ability to browse collections.

Homepage desktop mock up:
![Screen Shot 2020-03-06 at 2 44 06 PM](https://user-images.githubusercontent.com/6412857/76128404-1c809a00-5fb9-11ea-9bd5-62f799d70390.png)

Colors used:
Med. Blue background: #02779E at 75% opacity
Dark gray background: #575757 at  75% opacity
Text over both: #FFFFFF

Fold shown at blue marker / dotted line

### Background

Current homepage:
<img width="1371" alt="Screen Shot 2020-05-19 at 10 40 50 AM" src="https://user-images.githubusercontent.com/6412857/82359810-43961780-99bd-11ea-8fd7-b273047f8302.png">

### Acceptance Criteria

This is what done looks like

* [x] Change header on homepage to only include top dark blue bar
* [x] Finalize hero image to be used and get the image file
* [ ] Display thumbnail image for 8 "featured" collections that link out to the collection's landing page 
* [ ] "VIEW ALL COLLECTIONS" button links out to A-Z page of all collections

### Related Work 
Mock up ticket: #146 
[Mock up in Google Drive](https://drive.google.com/open?id=1jMHh-PiC9th5O2WSYMpUUb1cWWP5HYtE)

**Comment by JessWaggs** [comment]
@NedHenry Is there currently a way to designate a "featured" collection in some way? Or will this need to be a separate development ticket?

**Comment by NedHenry** [comment]
There is a "featured work" function that does exactly this for works, which we disabled. I don't know if an analogous "featured collection" function already built in, but if not it should be straightforward to replicate the "featured work" part. There may be some development work needed, but not a ton.

**Comment by rmjaffe** [comment]
During the current sprint, work on this ticket will focus on getting a better sense of contingencies as well as the actual scope of the work and time involved.  We may want to consider assigning a deadline for the remainder of the work once the investigation stage is complete.

**Comment by rschwab** [comment]
Contingencies/Dependencies:
1. Featured collections.
- This functionality does not yet exist. Use static content for now?
- Need to identify which collections we will start with.

2. View All Collections button.
- An A-Z collections page does not yet exist.
- Currently the homepage displays all public collections.
- This button could be a hide/show toggle to display the remaining collections while the A-Z page is being designed and built?

3. Banner image. 
- I need a high res version of the image of the person on the squiggle.
- The gradient can be added with css.

**Comment by chesleyperry** [comment]
@tmariamora and @katedundon  -  We're going to add 4 collections as "featured" collections on the DAMS home page.  Which 4 collections would you like to feature first?

**Comment by tmariamora** [comment]
SC County, UA50 and RMB/PJ. @katedundon what do you think about CMF? Part of me wants to feature but it might prove a disappointment for non-affiliates ;) Maybe Mayo if not CMF?

**Comment by JessWaggs** [comment]
Decision: Create homepage using static elements

1) JW will create a Featured Collections ticket for a future sprint (will not be a dependency for this ticket)
    SP will ask SCA to designate 4 collections for static homepage

2) Yes, toggle for static homepage

3) @chesleyperry Can you provide high res of the squiggle?

**Comment by katedundon** [comment]
I think Harry Mayo surfing photos probably make more sense as a featured collection over Cabrillo because it has visual appeal.

**Comment by rschwab** [comment]
@chesleyperry I tracked down the image here: https://digitalcollections.library.ucsc.edu/concern/works/rx913q84k?locale=en

I pulled the 1000px version to begin working with in the meantime.

**Comment by rschwab** [comment]
@JessWaggs would you like to pick images from these collections to be the thumbnail image? If not I can just grab what is currently on the collection landing pages.

**Comment by JessWaggs** [comment]
@rschwab Let's just use the current homepage/landing page thumbnails for now - thanks!

**Comment by chesleyperry** [comment]
Sorry it took me so long to find the hi res version of the squiggle:  https://drive.google.com/file/d/1xRd716BL0RwrqgMQZqHEs1a5tDmtUmA2/view?usp=sharing

**Comment by rschwab** [comment]
Pull request linked above. Screenshots:
![Firefox_Screenshot_2020-10-29T17-18-59 712Z](https://user-images.githubusercontent.com/515412/97611538-8da31080-19d3-11eb-9b9b-267445f57316.png)
![Firefox_Screenshot_2020-10-29T17-18-45 948Z](https://user-images.githubusercontent.com/515412/97611584-9b589600-19d3-11eb-8905-64f8178b4e79.png)

**Comment by JessWaggs** [comment]
@rschwab This is looking beautiful!
Two small aesthetic changes (and let me know if we can do these now or if they should be follow up tickets): 
1) could you left align the collection images, but space them out so that they don't look clumped to the left?
2) Could the collection images all be the same size/shape?

**Comment by chesleyperry** [comment]
If you need work on the images for Jess's suggestion 2, maybe @afrebert could help as our resident digital imaging specialist?

**Comment by rschwab** [comment]
Yeah that should be do-able on this one. I'll move this back to in-progress for now.

**Comment by rschwab** [comment]
New screenshot:
![Firefox_Screenshot_2020-10-30T20-54-02 413Z](https://user-images.githubusercontent.com/515412/97756411-0a0f1f80-1ab8-11eb-9405-8cbf82b3c8b1.png)

Images scale with screen size, and there are two sized versions for better loading on mobile.

**Comment by JessWaggs** [comment]
@rschwab This looks beautiful!

**Comment by chesleyperry** [comment]
I've QA'd both desktop and mobile and can find no issues.  I think it's ready.
One question - is there any way to have a separate URL with the "View all collections" button activated and all collections displayed?  The browse collections link in the header goes to the home page right now and it would be nice to have all collections displayed when you click on it.

**Comment by rschwab** [comment]
@chesleyperry I think the javascript could be amended to accomplish this. While it's not complicated, it's not exactly a quick fix either, so my vote would be to do this in a new ticket if we want to pursue it.

**Comment by JessWaggs** [comment]
Just to chime in that, eventually, both the button and the Browse Collections link will point to the A-Z page.

#277

**Comment by chesleyperry** [comment]
Since this is just an interim solution, it's probably not worth the time to amend the javascript.  It's likely a better idea to just prioritize the A-Z page, as Jess noted.
